scalameta/parsers/shared/src/main/scala/scala/meta/internal/parsers/ScalametaParser.scala

```diff
@@ -837,77 +840,6 @@ class ScalametaParser(input: Input, dialect: Dialect) { parser =>
       (isCaseClassOrObject || (token.is[KwCase] && token.next.is[Ident] && dialect.allowEnums))
   }
 
-  @classifier
```

Comment:
(stupid github only allows commenting on red/green lines) should we remove `isSoftKw` above?

Reply:
Ach right! Removed it!